sway-ir/src/module.rs

```diff
@@ -43,9 +43,31 @@ pub enum ConfigContent {
         encoded_bytes: Vec<u8>,
         decode_fn: Cell<Function>,
         opt_metadata: Option<MetadataIndex>,
+        indirect: bool,
```

Comment:
Nit: Using the name `indirect` creates a double negative - if `indirect = false` then you're saying "This buffer is not read indirectly", when it would be simpler to say "This buffer is read directly".

I would prefer if it was named `direct_read` or something similar, with `true` meaning "This buffer is read directly" and `false` meaning "This buffer is read indirectly".

Or maybe even scrap the direct/indirect terminology altogether and call it `read_from_offset` instead?

Reply:
Good idea. Done.
I am going with "storage" and an enum, so it is more clear.